src/components/ComingSoon.stories.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import ComingSoon from './ComingSoon'
+import { useCauseData } from 'src/utils/storybookHelpers/useCauseData'
+import { mobile, seas, cats } from '../../.storybook/boilerPlate'
+export default {
+  title: 'Pages/ComingSoon',
+  component: ComingSoon,
+}
+
```

Comment:
Will this handle a static "coming soon" countdown so there isn't UI review noise?

Reply:
Good point.